### ISSUE TITLE
Installation with conda - No module named 'Levenshtein'

### ISSUE THREAD

**Issue by evaegelyng** [adriantich/DnoisE#13]
When installing DnoisE with conda, there seems to be a problem locating the Levenshtein module ("ModuleNotFoundError: No module named 'Levenshtein')? In my case, this could be fixed by reinstalling python-levenshtein using pip3, but maybe you could check this issue? 

Best,

Eva

**Comment by adriantich** [comment]
Hello Eva,
We have now changed some things of the code.
The Levenshtein package that we used from https://github.com/ztane/python-Levenshtein is not under mantainament so we have changed it to https://maxbachmann.github.io/Levenshtein/. 
However, during this week we are still working on the new version of DnoisE that will be released soon. If you clone the DnoisE from the github repository, should work properly. If you prefer the anaconda installation, it will be ready next week hopefully.

Thanks for your feedback and sorry for the delay. We had to fix some bugs and issues.

Adrià
